src/order/create-order.ts

```diff
@@ -11,22 +11,38 @@ const {
   paypal: { apiBaseUrl },
 } = config;
 
-type CreateOrderResponse = {
+type HTTPStatusCodeSuccessResponse = 200 | 201;
```

Comment:
```suggestion
type HTTPStatusCodeSuccessResponse = 200 | 201 | 204 | 207;
```
More success codes returned by v2/checkout/orders endpoints

Reply:
Good point about 204 and 207. I'll include those with the other v2 orders api calls. For createOrder() I'm going to keep it as 200 and 201 only since those are the only valid status codes I've seen return from it.